### PR TITLE
Windows setup

### PR DESCRIPTION
**Pull request by niccokunzmann**
Add a setup for Windows #58 
Sadly it is wihtout visual studio. If you like, you can use visual studio to edit the files.

### CODE REVIEW

**Comment by niccokunzmann** [comment]
@Charle42 Would you like to merge?

File: dev/_posts/2017-05-18-setup-schul-cloud-client-and-sever-under-windows.md [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+.version" }
+2017-05-18T11:45:54.545+0200 I INDEX    [initandlisten]          building index using bulk method; build may temporarily use up to 500 megabytes of RAM
+2017-05-18T11:45:54.547+0200 I INDEX    [initandlisten] build index done.  scanned 0 total records. 0 secs
+2017-05-18T11:45:54.548+0200 I COMMAND  [initandlisten] setting featureCompatibilityVersion to 3.4
+2017-05-18T11:45:54.549+0200 I NETWORK  [thread1] waiting for connections on port 27017
+```
+
+Now, the database is running.
+
+In an other console window, we need to log in and create the database for the server.
```
Comment (Langleu) [inline dev/_posts/2017-05-18-setup-schul-cloud-client-and-sever-under-windows.md]: and once again I ask you what for do you create an user?
Following your tutorial you did not have any intentions of starting mongod in authentication mode --> creating user --> false hope of security.
It makes no sense to create an user if you don't intend to actually expose your database to the world and if then you should have enough knowledge of the security risks etc....

File: dev/_posts/2017-05-18-setup-schul-cloud-client-and-sever-under-windows.md [hunk 2]
```diff
@@ -0,0 +1,372 @@
...
+Here, you can switch between the repositories.
+![]({{ images }}/04-06-clone-repo.png)
+![]({{ images }}/04-07-clone-repo.png)
+
+
+## Install the Packages (Server)
+
+![]({{ images }}/05-01-install-packages.png)
+
+The command promt opens.
```
Comment (Langleu) [inline dev/_posts/2017-05-18-setup-schul-cloud-client-and-sever-under-windows.md]: prompt

File: dev/_posts/2017-05-18-setup-schul-cloud-client-and-sever-under-windows.md [hunk 3]
```diff
@@ -0,0 +1,372 @@
...
+2. Accept license agreement
+3. let the defaults be like they are
+   ![]({{ images }}/02-03-install-node-js.png)
+   
+
+## Install Git
+
+Git is the version control system (VCS) we use to exchange the source code
+and write several versions of it.
+All the Schul-Cloud repositotries are hosted on [Github](https://github.com/schul-cloud).
```
Comment (Langleu) [inline dev/_posts/2017-05-18-setup-schul-cloud-client-and-sever-under-windows.md]: repositories

File: dev/_posts/2017-05-18-setup-schul-cloud-client-and-sever-under-windows.md [hunk 4]
```diff
@@ -0,0 +1,372 @@
...
+```
+C:\Users\username>dbseed.bat
+```
+
+Note: if this file is not present, you can get it from [here](https://github.com/schul-cloud/schulcloud-server/pull/165).
+
+
+## Start the Server
+
+Make sure that the database is runnning.
```
Comment (Langleu) [inline dev/_posts/2017-05-18-setup-schul-cloud-client-and-sever-under-windows.md]: running

**Comment by Langleu** [comment]
throughout the whole text I see over and over again proper nouns written in 10 different styles.
It would be nice to have those written in a consistent way

File: dev/_posts/2017-05-18-setup-schul-cloud-client-and-sever-under-windows.md (hunk 1 above)
Comment (niccokunzmann) [inline dev/_posts/2017-05-18-setup-schul-cloud-client-and-sever-under-windows.md]: So, you suggest, the section can be removed?

**Comment by niccokunzmann** [comment]
@Langleu Thanks fo rthe review. I am on it.

**Comment by niccokunzmann** [comment]
@Langleu Would you like to look at https://github.com/schul-cloud/schul-cloud.github.io/pull/59/commits/ff7c95ba0dba4baddadb63915c9317d5d4dd21ba to make sure, I did not delete too much?

**Pull request merged**